src/AssetResponder.php

```diff
@@ -118,7 +118,6 @@ protected function ok()
         return $response
             ->withBody(new CallbackStream($callable))
             ->withHeader('Content-Length', (string) filesize($this->data->asset->path))
-            ->withHeader('Content-Type', $this->data->asset->type)
```

Comment:
This is  a purposeful one. Can you please revert this ?

The idea is, it helps to comment any code without looking for the `;` at the end. `ctrl + / ` else we need to and put the `;` some where else ;-) .

Reply:
urg, ok...